contrib/cirrus/runner.sh

```diff
@@ -28,6 +28,15 @@ _run_validate() {
     make validate
 }
 
+_run_verify_vendor() {
+   if git diff --no-ext-diff --quiet --exit-code;
+   then
+     echo No tampering with vendor...;
+   else
+     return 1;
+   fi;
+}
+
```

Comment:
Not-blocking:  I would prefer the opposite (unix-like) behavior - be silent on success and make noise on failure.  Might I suggest something like:

```bash
_run_verify_vendor() {
    # N/B: current repo. dir. contents produced by _run_build() above.
    if ! git diff --no-ext-diff --quiet --exit-code; then
        die "Found uncommited and necessary changes to vendoring, please fix, commit, and re-submit."
    fi
}
```

P.S. No need for semi-colons in bash unless multiple statements are on one line.  I presume rust has scrambled your brain, so I forgive you :grinning:

Reply:
:smile: yeah ! Fixed it now thanks :)